guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -14,14 +14,17 @@
     "# Table Of Contents\n",
```

Comment:
<p>the part should be moved down after model initialization</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>moved down after model initialization</p>